src/block_cache/block_chunkcache.c

```diff
@@ -881,9 +881,12 @@ __wt_chunkcache_get(WT_SESSION_IMPL *session, WT_BLOCK *block, uint32_t objectid
                  * Increment the access count for eviction. If we are accessing the new chunk, the
                  * access count would have been incremented on it when it was newly inserted to
                  * avoid eviction before the chunk is accessed. So we are giving two access counts
-                 * to newly inserted chunks.
```

Comment:
Commenting here since I can only comment on so much of the diff. You mentioned on the ticket that limiting the amount of content evicted per round wasn't in scope for this ticket. That's fine, but two questions:

1. We were talking about stopping eviction once the chunk cache fullness level had gone below the eviction target. Is that covered in the statement above?
2. From memory, the chunk cache hang in test/format was caused by excessive eviction. Is that still an issue after this PR?

Reply:
1. Considering the eviction logic, I believe it is not ideal to halt eviction when we reach the target threshold, as this would essentially leave us hovering near the target threshold, strongly promoting the eviction of recently accessed chunks (since they would typically have lower access counts compared to chunks already in the cache). Instead,  a more suitable approach could involve restricting eviction to approximately 15% of the total chunks per round.
2. Thanks for reminding me of this, I have run a patch to test the same.

Reply:
<img width="1476" alt="Screenshot 2023-12-04 at 1 41 50 pm" src="https://github.com/wiredtiger/wiredtiger/assets/55784503/9e1014da-3746-424d-aed2-117e97fe5e6c">
Ran format and the eviction rate looks quite consistent, with no mass eviction. I have run a patch with 100 iterations.